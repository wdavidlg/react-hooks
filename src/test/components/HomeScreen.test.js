const { mount } = require("enzyme")
const { default: HomeScreen } = require("../../components/09-useContext/HomeScreen");
const { default: UserContext } = require("../../components/09-useContext/UserContext");


describe('Pruebas con <HomeScreen />', () => {

    test('debe de mostrarse correctamente', () => {

        const user = {name: 'will', email: 'will@will.com'};

        const wrapper = mount(
          <UserContext.Provider value={{
            user
          }}>
              <HomeScreen/>
          </UserContext.Provider>  
        );
        expect(wrapper).toMatchSnapshot();
    })
    
    
})
