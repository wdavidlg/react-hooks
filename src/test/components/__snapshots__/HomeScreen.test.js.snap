// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas con <HomeScreen /> debe de mostrarse correctamente 1`] = `
<div>
  <h2>
    HomeScreen
  </h2>
  <hr />
  <pre>
    {
    "name": "will",
    "email": "will@will.com"
}
  </pre>
</div>
`;
